WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -947,8 +947,17 @@ private void LogCoinJoinSummary(ImmutableArray<AliceClient> registeredAliceClien
 	private static double GetAnonLoss<TCoin>(IEnumerable<TCoin> coins)
 		where TCoin : ISmartCoin
 	{
+		if (coins.Count() <= 1)
+		{
+			return 0;
+		}
+
+		// Parameters were picked experimentally to model anonymity loss that matches real-world experience.
```

Comment:
Reference your PR. It's very well explained, it should be useful for anyone looking into it in the future:
```suggestion
		// Parameters were picked experimentally to model anonymity loss that matches real-world experience: https://github.com/zkSNACKs/WalletWasabi/pull/10096
```